### DISCUSSION TITLE
"Your GitHub academic discount coupon has expired" message won't go away

### DISCUSSION THREAD

**Discussion by Resonious** [community/community#21660] [New to GitHub]
<p>My academic discount is expiring. All according to plan. I’m no longer a student, so no problem. I’ve opted to downgrade my account, but the message still appears on every single GitHub page. It’s starting to drive me nuts. Is there a way I can get rid of it, or am I stuck with it until my downgrade goes through (in a couple weeks)?</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<blockquote>
<p>Is there a way I can get rid of it, or am I stuck with it until my downgrade goes through (in a couple weeks)?</p>
</blockquote>
<p>That depends! Do you need the features of the paid plan right now? If not, I can push through your downgrade so it goes through immediately.</p>

**Comment by Resonious** [comment]
<p>I’m not using any pro features right now, so if you could expedite the downgrade that’d be great! Thanks for the quick response!</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:fairy: Try that now!</p>

**Comment by noah-johnson** [comment]
<p><a class="mention" href="/canuckjacq">@canuckjacq</a><br>
could i have this done for my account as well? the banner is enormous and on every single page of github</p>

**Comment by canuckjacq** [comment]
<p>Sorry <a class="mention" href="/noah-johnson">@noah-johnson</a> I didn’t see this until now.</p>
<p>If you didn’t get it sorted already, please let me know.</p>

**Comment by patoroco** [comment]
<p>Hi,<br>
I’m having the same issue after finishing my academic discount, could I have this done for my account as well?</p>
<p>Thanks!</p>

**Comment by canuckjacq** [comment]
<p>that’s done for you now!</p>

**Comment by patoroco** [comment]
<p>oh, it was fast! Thank you :slight_smile:</p>

**Comment by yildirayunlu** [comment]
<p>Hi,<br>
I’m having the same issue after finishing my academic discount, could I have this done for my account as well?</p>
<p>Thanks!</p>

**Comment by canuckjacq** [comment]
<p><a class="mention" href="/yildirayunlu">@yildirayunlu</a> that’s done!</p>

**Comment by yildirayunlu** [comment]
<p>Thank you <a class="mention" href="/canuckjacq">@canuckjacq</a> :pray:t3:</p>

**Comment by Itay4** [comment]
<p><a class="mention" href="/canuckjacq">@canuckjacq</a> im having the same issue, can you assist please?</p>

**Comment by canuckjacq** [comment]
<p><a class="mention" href="/itay4">@Itay4</a> that’s done for you now!</p>

**Comment by Itay4** [comment]
<p>cool thanks so much !</p>

**Comment by suyogxda** [comment]
<p>Hi <a class="mention" href="/canuckjacq">@canuckjacq</a>, I’m having the same issue, the banner doesn’t go away. Can you help me too?</p>

**Comment by canuckjacq** [comment]
<p>Hi - you haven’t actually downgraded so the account is meant to be locked. If you <a href="https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-billing-and-payments-on-github/downgrading-your-github-subscription#downgrading-your-user-accounts-subscription">downgrade your account</a> and you’re still seeing that banner, let me know.</p>

**Comment by prateekiiest** [comment]
<p>Hi, I am facing the same issue. I have downgraded already but the banner is still showing up.</p>

**Comment by canuckjacq** [comment]
<p><a class="mention" href="/prateekiiest">@prateekiiest</a> that should be gone now!</p>

**Comment by JPinkney** [comment]
<p>I’m running into the same issue as well! Can this be done for me please?</p>

**Comment by bartimusprimed** [comment]
Can you remove mine as well please.

**Comment by appledragon** [comment]
I’m running into the same issue as well! Can this be done for me please?

**Comment by meznader** [comment]
i tried to use data camp This discount has expired or cannot be applied to your current plan. any help please

**Comment by meznader** [comment]
i tried to get 3 months datacamp but i got this error message :This discount has expired or cannot be applied to your current plan.
![image](https://github.com/community/community/assets/162059171/24b983dc-600a-4efc-a4e9-74632348799e)

**Comment by isyuricunha** [comment]
@meznader , usually this happens if you have already made any offer in your account.
if i'm not mistaken, if you have already tried any kind of trial period, you cannot get another one elsewhere, as they also do not add up time.

you'll have to check this directly with datacamp.

edit: maybe you can check your history plan/bills/invoices, some business show the trial time

**Comment by isyuricunha** [comment]
said that, i believe this is not the correct topic.

**Comment by cadsdfsfe** [comment]
I’m running into the same issue as well! Can this be done for me please?

**Comment by mrjavadseydi** [comment]
i have the same issue , it's stuck on pending downgrade for days. 
<img width="800" height="180" alt="Screenshot 2026-04-16 at 10 42 00 PM" src="https://github.com/user-attachments/assets/c4f67df0-1263-472f-8c48-cd8a99b49da7" />

can someone help me ?
